build_tools/azure/test_script.sh

```diff
@@ -11,7 +11,7 @@ if [[ "$BUILD_REASON" == "Schedule" ]]; then
     # Enable global random seed randomization to discover seed-sensitive tests
     # only on nightly builds.
     # https://scikit-learn.org/stable/computing/parallelism.html#environment-variables
-    export SKLEARN_TESTS_GLOBAL_RANDOM_SEED="any"
+    export SKLEARN_TESTS_GLOBAL_RANDOM_SEED=$(($RANDOM % 100))
```

Comment:
I like this alternative, it's a lot more intuitive.

Reply:
@lesteve Can we do a quick "echo" here to show `SKLEARN_TESTS_GLOBAL_RANDOM_SEED` in the CI output?

Reply:
We can even move the message of the header here

Reply:
Done